bin/deactivate.bat

```diff
@@ -1,36 +1,36 @@
 @echo off
+setlocal
 
-for /f "delims=" %%i in ("%~dp0..\envs") do (
-    set ANACONDA_ENVS=%%~fi
-)
-
-if "%1" == "" goto skipmissingarg
+if not "%1" == "--help" goto skipusage
+    (
     echo Usage: deactivate
     echo.
     echo Deactivates previously activated Conda
     echo environment.
+    ) 1>&2
+    exit /b 1
+:skipusage
+
+if "%1" == "" goto skiptoomanyargs
```

Comment:
This has tripped up some people who try to pass env names to deactivate.  I'm a little mixed on whether it is necessary at all.  If arguments are provided, would it be OK just ignore them rather than raising an error?  Perhaps show a warning that tells them that passing env name is superfluous, since they can only deactivate the active env?